docs/AutomationReference.md

```diff
@@ -280,12 +292,25 @@ These versions embed the symbols into the assemblies and support [SourceLink](ht
 
 #### Error handling
 
-`AxeWindowsAutomationException` is thrown for all unhandled errors in Axe.Windows.Automation. If an exception was thrown from code not owned by Axe.Windows.Automation, that exception will be wrapped in the `Exception.InnerException` property.
+`AxeWindowsAutomationException` is thrown for errors in `Axe.Windows.Automation`'s logic where meaningful error reporting can be generated. In some situations, the `Exception.InnerException` property may contain a corresponding system-level exception for errors encountered by Axe.Windows.
+
+#### Migrating from Axe.Windows 1.x
+Migration to Axe.Windows 2.0 from 1.x should require minimal code changes for most projects.
+
+##### `IScanner.ScanAll` removed
+The `ScanAll` method for scanning multiple top-level windows has been removed. Use `ScanAsync` or `Scan` instead.
```

Comment:
This brings up an interesting point that I didn't consider before. Scanning multiple-level windows was a two-step process--the user had to enable multiple scans on the ConfigBuilder, then call the ScanAll API. That was for backward compatibility. Since 2.0 is a breaking change, maybe we can revisit that. What are your thoughts on enabling multiple top-level windows by default?

Reply:
The change to always enable multiple top-level windows has been merged